teku/src/main/java/tech/pegasys/teku/cli/subcommand/storage/RocksDbCommand.java

```diff
@@ -0,0 +1,103 @@
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.cli.subcommand.storage;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.rocksdb.RocksDBException;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Mixin;
+import tech.pegasys.teku.cli.converter.PicoCliVersionProvider;
+import tech.pegasys.teku.cli.options.BeaconNodeDataOptions;
+import tech.pegasys.teku.cli.options.Eth2NetworkOptions;
+import tech.pegasys.teku.infrastructure.logging.SubCommandLogger;
+import tech.pegasys.teku.networks.Eth2NetworkConfiguration;
+import tech.pegasys.teku.service.serviceutils.layout.DataDirLayout;
+import tech.pegasys.teku.storage.server.rocksdb.RocksDbHelper;
+
+@Command(
+    name = "rocksdb",
+    description = "Print RocksDB information",
+    mixinStandardHelpOptions = true,
+    versionProvider = PicoCliVersionProvider.class)
+public class RocksDbCommand implements Runnable {
+
+  @Override
+  public void run() {
+    CommandLine.usage(this, System.out);
+  }
+
+  public static final SubCommandLogger SUB_COMMAND_LOG = new SubCommandLogger();
+
+  @Command(
+      name = "usage",
+      description = "Print disk usage",
+      mixinStandardHelpOptions = true,
+      versionProvider = PicoCliVersionProvider.class)
+  public int usage(
+      @Mixin final BeaconNodeDataOptions beaconNodeDataOptions,
+      @Mixin final Eth2NetworkOptions eth2NetworkOptions) {
+
+    RocksDbHelper.printTableHeader(SUB_COMMAND_LOG);
+    final String dbPath =
+        DataDirLayout.createFrom(beaconNodeDataOptions.getDataConfig()).getBeaconDataDirectory()
+            + "/db";
+    final Eth2NetworkConfiguration networkConfiguration =
+        eth2NetworkOptions.getNetworkConfiguration();
+    final List<RocksDbHelper.ColumnFamilyUsage> columnFamilyUsages = new ArrayList<>();
+    RocksDbHelper.forEachColumnFamily(
+        dbPath,
+        networkConfiguration,
+        (rocksdb, cfHandle, spec) -> {
+          try {
+            columnFamilyUsages.add(
+                RocksDbHelper.getAndPrintUsageForColumnFamily(
+                    rocksdb, cfHandle, spec, SUB_COMMAND_LOG));
+          } catch (RocksDBException e) {
+            throw new RuntimeException(e);
+          }
+        });
+    RocksDbHelper.printTotals(SUB_COMMAND_LOG, columnFamilyUsages);
+    return 0;
+  }
+
+  @Command(
+      name = "stats",
+      description = "Print rocksdb stats",
+      mixinStandardHelpOptions = true,
+      versionProvider = PicoCliVersionProvider.class)
```

Comment:
this one is a whole hard thing to read, Im wondering if something like a column filter or something would be useful...

Reply:
Another option is print a better header rather than the in-line which doesnt really make anything at all clear...

Reply:
hrm maybe i was reading it wrongly, this isnt so bad...
```
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Column Family: FINALIZED_STATES_BY_SLOT
```

Reply:
Can we add this filter on a separate PR? I don't think this is crucial for what I'm trying to get this PR merged in.

Reply:
thats fine, the history is just so long its not usable and you need to redirect to file.